### DISCUSSION TITLE
For an outsider the project looks stale

### DISCUSSION THREAD

**Discussion by Geegee** [syl20bnr/spacemacs#16586] [Q&A] [answered]
I love this project very much and am using it for many years now. I several times tried to
get in contact with people using it ( just hanging out) but when joining irc, gitter chat and so on it seems 
very inactive.

When going on the spacemacs web presence and hit the news button I am guided to a page
that mentions 2016 as a major step. 

Yet there seems to be development going on when I check the commit history.

Another minor  thing is that  I can't seem to find a way to donate to the project besides the 
shop. I just want to give money and don't buy more useless things I don't need.

**Comment by bryce-carson** [comment, marked as answer]
> I love this project very much and am using it for many years now. I several times tried to get in contact with people using it ( just hanging out) but when joining irc, gitter chat and so on it seems very inactive.

Much of the user-base does not use the Gitter chat or IRC at all, especially the more self-sufficient users. The documentation promotes self-study, as does Emacs itself. We don't have a Discord, which I think would be more popular but it is also a good thing we don't because the instant messaging form of communication promotes people to act silly and demand instant reply!

> When going on the spacemacs web presence and hit the news button I am guided to a page that mentions 2016 as a major step.
> 
> Yet there seems to be development going on when I check the commit history.

Indeed, the Spacemacs website documents a master branch that hasn't been updated in a long time. Everyone is on the develop branch and there are no plans to have another "stable" release, because it's not realistic to try that. Emacs itself can do that because it's configuration is nil, but for Spacemacs there are too many moving parts to have any sort of "stable" release with patches applied to it coming from a develop branch.

> Another minor thing is that I can't seem to find a way to donate to the project besides the shop. I just want to give money and don't buy more useless things I don't need.

Sylvian B. (syl20bnr) can be supported using GitHub Sponsors, which helps cover the costs of the domain registration for the Spacemacs website. It's up to you to whether you want to do that. You could also use GitHub Sponsors to support one of the active maintainers (@smile13241324).

**Reply by Geegee** [reply]
Sorry for the late reply. Thank you very much for taking the time to answer. I will give then a sponsorship
to one of the maintainers then. Thanks to everyone for all the incredible work.
